### PR TITLE
Literature

### PR DESCRIPTION
**Pull request by ThierryO**
- Rename the main topic from `other resources` into `literature`
- Split the `further reading` into several topics
- Add new content based on the bibtex provided by @hansvancalster 

The page with literature on statistics still needs work. For the time being, it is just a framework to display the annotations or abstract from the bibtex file.

### CODE REVIEW

**Comment by ThierryO** [comment]
I think that `statistics.Rmd` could use a bit more structure and an introduction. Maybe the order of the books needs some tweaking too.

The other pages might need an updated introduction as they all have the same original introduction.

All `Rmd` files in `literature` still need a `description` in the yaml header. This description will be displayed at literature overview page. The order of the different topics can be controlled by setting a `weight` argument in the yaml. Pages with a higher weight will be displayed first.

File: content/literature/computing.Rmd [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+by Julia Lowndes at the SAFRED conference, Brussels, 27 Feb 2018.
+
+## Overviews {#overviews}
+
+- @wilson_good_2017: set of **good computing practices** that every researcher can adopt
+- @british_ecological_society_guide_2014: planning the **data life cycle**; creating, processing, documenting, preserving, sharing & reusing data
+- @cooper_guide_2017: file organisation, workflow documentation, **code** reproducibility and readability, writing reproducible reports, version control and code archiving
+- @ibanez_practicing_2014: vision on reproducible science, routine **practices**, collaboration, literate computing
+
+See also some resources related to [learning and education](#educate) and the [discipline of open and reproducible science](#discipline).
```
Comment (florisvdh) [inline content/literature/computing.Rmd]: The `#educate` and `#discipline` anchors should be reinserted in `open_science.Rmd` and `skills.Rmd`.

File: content/literature/skills.Rmd [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+bibliography: reproducible_research.bib
+---
+
+This page lists **selected** literature and online resources. Some are related to existing tutorial pages, while others are not. They are supposed to be of high interest to this site's users.
+
+Several of the resources were added based on an inspiring
+[talk](https://docs.google.com/presentation/d/10KkXEv4r3wWtdKvB6RFOPe809eMNldODaRqQxn-jeME/edit?usp=sharing) 
+by Julia Lowndes at the SAFRED conference, Brussels, 27 Feb 2018.
+
+These resources focus on the learning and teaching aspect, while they often also give an overview of scientific computing workflows (as [above](#overviews)).
```
Comment (florisvdh) [inline content/literature/skills.Rmd]: replace 'above' by 'before'

File: content/literature/open_science.Rmd [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+bibliography: reproducible_research.bib
+---
+
+This page lists **selected** literature and online resources. Some are related to existing tutorial pages, while others are not. They are supposed to be of high interest to this site's users.
+
+Several of the resources were added based on an inspiring
+[talk](https://docs.google.com/presentation/d/10KkXEv4r3wWtdKvB6RFOPe809eMNldODaRqQxn-jeME/edit?usp=sharing) 
+by Julia Lowndes at the SAFRED conference, Brussels, 27 Feb 2018.
+
+These resources focus on the discipline as a whole, its perception, principles ,etc., while they often also give an overview of scientific computing workflows (as [above](#overviews)).
```
Comment (florisvdh) [inline content/literature/open_science.Rmd]: replace 'above' by 'before'

Comment (ThierryO) [reply content/literature/open_science.Rmd]: solved in a4dace0e

File: content/literature/computing.Rmd (hunk 1 above)
Comment (ThierryO) [reply content/literature/computing.Rmd]: solved in a4dace0e

File: content/literature/skills.Rmd (hunk 2 above)
Comment (ThierryO) [reply content/literature/skills.Rmd]: solved in a4dace0e

**Pull request merged**